src/styles.scss

```diff
@@ -103,3 +103,15 @@ html {
   clip: rect(0, 0, 0, 0);
   border: 0;
 }
+
```

Comment:
Cami we have a folder `src/assets/scss/material.orcid.overwrites.scss` to put all material overwrites of the app. 

Let's put these overwrites there!

Reply:
created a new file form-field.css for the placeholders overwrites